packages/store/src/useBookmarkStore.ts

```
@@ -5,19 +5,24 @@ import {
   DeleteBookmarkPayload,
+  recentBookmark,
   UpdateBookmarkPayload,
 } from "@repo/types";
 import { useFolderStore } from "./useFolderStore";
 
 interface BookmarkStore {
   bookmarks: Bookmark[];
+  notes: Bookmark;
+  recentBookmarks: recentBookmark[];
   fetchBookmarks: (folderId: string) => Promise<void>;
+  fetchNotes: (id: string) => Promise<Bookmark | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)